docs/source/geopmwrite.1.rst

```diff
@@ -122,6 +129,12 @@ Options
                 permissions no operation will be performed.  To force the
                 creation of a new cache file, remove the existing cache file
                 prior to executing this command.
+-f, --config    Read control name, control domain, control index and control value
+                from a configuration file rather than using the positional
+                arguments.  These four parameters is provided on each line of
+                the file separated by white space.  The file may have many lines
+                specifying multiple controls to be written. Proving ``-`` for this
+                option specifies to read the configuration from standard input.
```

Comment:
```suggestion
-f, --config    Read control name, control domain, control index and control value
                from a configuration file rather than using the positional
                arguments.  These four parameters are provided on each line of
                the file separated by white space.  The file may have many lines
                specifying multiple controls to be written. Providing ``-`` for this
                option specifies to read the configuration from standard input.
```